cookbooks/aws-parallelcluster-config/templates/default/directory_service/sssd.conf.erb

```diff
@@ -0,0 +1,38 @@
+[domain/default]
+debug_level = 0x1f0
+id_provider = ldap
+cache_credentials = True
+ldap_schema = AD
+ldap_uri = <%=  node['cluster']["directory_service"]["domain_addr"] %> # TODO: use ldaps
+ldap_search_base = <%=  node['cluster']["directory_service"]["domain_name"] %>
+ldap_default_bind_dn = <%=  node['cluster']["directory_service"]["domain_read_only_user"] %> # TODO: parametrize username
+ldap_default_authtok_type = obfuscated_password
+ldap_default_authtok = <%= @obfuscated_password %>
+ldap_tls_cacert = /etc/openldap/certs/privatekey.pem
+ldap_tls_reqcert = never # TODO: use 'require'
```

Comment:
there are cookbook attributes for these two right?

Reply:
Done